students/sikorski_marcin/Lesson_1_-_Python_basics.py

```diff
@@ -0,0 +1,128 @@
+#1 Sum of three numbers
+
+a = int(input())
```

Comment:
Część nazw zmiennych jest jedno literkowa, przez co jest nie czytelna (jedynie h,m,s przy czasie da się domyśleć bo jest to często używane).
Co prawda szablony Snakify często tak sugerują, ale to nie znaczy, że to jest dobre podejście.